internal/providers/terraform/azure/testdata/storage_account_test/storage_account_test.golden

```diff
@@ -89,56 +89,153 @@
+ ├─ List and create container operations                         100  10K operations                   $5.50 
+ ├─ Read operations                                               10  10K operations                   $0.10 
+ ├─ All other operations                                         100  10K operations                   $0.44 
+ ├─ Data retrieval                                             1,000  GB                              $10.00 
+ ├─ Blob index                                                    10  10K tags                         $0.39 
+ └─ Early deletion                                             1,000  GB                              $15.20 
+                                                                                                             
+ azurerm_storage_account.v2_cool_lrs_nfsv3                                                                   
+ ├─ Capacity                                               1,000,000  GB                          $15,000.00 
+ ├─ Iterative Write operations                                 1,700  100 operations                 $221.00 
+ ├─ Write operations                                             100  10K operations                  $13.00 
+ ├─ Iterative Read operations                                     15  10K operations                   $1.08 
+ ├─ Read operations                                               10  10K operations                   $0.13 
+ ├─ All other operations                                         100  10K operations                   $0.60 
+ ├─ Data retrieval                                             1,000  GB                              $10.00 
+ └─ Early deletion                                             1,000  GB                              $15.00 
+                                                                                                             
+ azurerm_storage_account.v2_cool_ragzrs                                                                      
+ ├─ Read operations                                               10  10K operations                   $0.10 
+ ├─ All other operations                                         100  10K operations                   $0.44 
+ └─ Data retrieval                                             1,000  GB                              $10.00 
+                                                                                                             
+ azurerm_storage_account.v2_hot_gzrs                                                                         
+ ├─ Read operations                                               10  10K operations                   $0.04 
+ └─ All other operations                                         100  10K operations                   $0.44 
+                                                                                                             
+ azurerm_storage_account.v2_hot_lrs                                                                          
```

Comment:
I'm seeing a lot of usage data defined for this, but not the cost components to match:
```
    data_at_rest_storage_gb: 10000
    early_deletion_gb: 1000
    snapshots_storage_gb: 10000
    metadata_at_rest_storage_gb: 10000
    storage_gb: 1000000
    monthly_iterative_write_operations: 170000
    monthly_write_operations: 1000000
    monthly_list_and_create_container_operations: 1000000
    monthly_iterative_read_operations: 150000
    monthly_read_operations: 100000
    monthly_other_operations: 1000000
    monthly_data_retrieval_gb: 1000
    monthly_data_write_gb: 1000
    blob_index_tags: 100000
 ```
 e.g. I'd be expecting a `Metadata at rest` and `Data at rest` cost component.

Reply:
Metadata at rest and Data at rest are relevant only to `FileStorage`, Same as `Iterative X` components are available to GPv2. I can remove them from the examples where it's not applicable. I guess it was my copypasta.

Reply:
Ah okay, that makes sense. Yeah I'd remove them from the usage file for this test.